test/functional/api/v1/dependencies_controller_test.rb

```diff
@@ -0,0 +1,125 @@
+require 'test_helper'
+
+class Api::V1::DependenciesControllerTest < ActionController::TestCase
+  def self.should_respond_to(format)
+    should "return #{format.to_s.upcase} with the dependencies" do
+      @rubygem = create(:rubygem, name: "rack")
+      @version = create(:version, number: "1.0.0", rubygem_id: @rubygem.id)
+      get :index, gems: "rack", format: format
+    end
+  end
+
+  ## JSON ENDPOINTS:
+  # NO GEMS:
+  context "On GET to index --> with no gems --> JSON" do
+    setup do
+      @rubygem = create(:rubygem, name: "testgem")
+      @version = create(:version, number: "1.0.0", rubygem_id: @rubygem.id)
+      get :index, gems: "", format: "json"
+    end
+
+    should "return 200" do
+      assert_response :success
+    end
+
+    should_respond_to(:json) do |body| # (Should have an empty body)
+      body.should eq("")
```

Comment:
this is still unaddressed -- https://github.com/rubygems/rubygems.org/pull/1225/files#diff-353f6c78769d5197b05710e4547b0998R4 doesn't yield